parser/sql/dialect/mysql/src/main/antlr4/imports/mysql/DMLStatement.g4

```diff
@@ -287,7 +287,10 @@ tableReference
     ;
 
 tableFactor
-    : tableName partitionNames? (AS? alias)? indexHintList? | subquery AS? alias (LP_ columnNames RP_)? | LP_ tableReferences RP_
```

Comment:
Do you think we should refactor antlr parse logic accord to MySQL yacc?

Reply:
Aha, it was only after going through all the rules of table_factor in MySQL yacc that I finally understood how to modify it. If want to have a better reading experience and easily refer back to MySQL yacc for future modifications, I believe refactoring is necessary. What do you think, mentor? If we need to refactor, I will start immediately.

Reply:
I think this priority can be slightly lower, you can try to handle the unsupported sql parsing first.

Reply:
In addition, can you send your contact information to my email address? @zihaoAK47

Reply:
Of course, does the contact information use WeChat, phone number, or Twitter

Reply:
I think wechat is better.

Reply:
Has been sent to your email